libraries/default.rb

```diff
@@ -40,6 +40,66 @@ def self.error_message(error_code)
     end
   end
 
+  class ZookeeperConfig
```

Comment:
it looks like your class is very similat to a hash (the name of its method are extremely similar, maybe you could inherit from hash?)

Reply:
I'm trying to do it. I chose an array in the first place because I was unsure I could manage to keep ordering with a hash, but it appears ruby hashes are ordered.